platform/ui/src/components/Tooltip/PortalTooltip.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+import Card from './PortalTooltipCard';
+
+const portalNodes = {};
+
+/**
+ * A portal based tooltip.
```

Comment:
React portals sound cool - until now I did not know about them. Thanks for thinking of this. However I am bit surprised that we need to use the older style class type components. I did a quick search of the code base and noticed that this would be the first/only component that uses the old style - but I might be wrong. Are we actually forced to use this? I did see this https://react.dev/reference/react-dom/createPortal#rendering-to-a-different-part-of-the-dom, does it help so that we can use a function component instead?

Reply:
> React portals sound cool - until now I did not know about them. Thanks for thinking of this. However I am bit surprised that we need to use the older style class type components. I did a quick search of the code base and noticed that this would be the first/only component that uses the old style - but I might be wrong. Are we actually forced to use this? I did see this https://react.dev/reference/react-dom/createPortal#rendering-to-a-different-part-of-the-dom, does it help so that we can use a function component instead?

I repurposed this component. It's not exposed and not worth moving the trouble migrating it to function component. I added it just to workaround this specific issue, we should refactor the left panel so we can go back and use the original tooltip instead. Also, this is my last effort to address this specific issue without touching the left panel.